.github/CODEOWNERS

```diff
@@ -46,3 +46,10 @@ pnpm-lock.yaml
 
 # i18n Quotes
 **/motivation.json @freeCodeCamp/staff @freeCodeCamp/i18n
+
+# -------------------------------------------------
+# Files that need attention from the mobile team
+# -------------------------------------------------
+
+/curriculum/schema/challenge-schema.js @freeCodeCamp/mobile
+/client/src/redux/prop-types.ts @freeCodeCamp/mobile
```

Comment:
Is this just a hack? dev-team should already be requested because of `/* @freecodecamp/dev-team`

I'll try it regardless, but I'm curious about the rationale.

Reply:
Unless specified explicitly, the change would have become an override and only the mobile team would be review requested.

Reply:
Ah, thanks for explaining!